CSS/style.css

```diff
@@ -1,3 +1,1444 @@
+  border-top-left-radius: 80px;
+  position: relative;
+  box-sizing: border-box;
+  margin-top: -85px;
+}
+
+.contact-header {
+  width: 375px;
+  height: 56px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #fff;
+  margin-top: 88px;
+  margin-bottom: 0;
+}
+
+textarea {
+  margin-bottom: 35px;
+  resize: none;
+  border: 0;
+}
+
+.contact-desc {
+  width: 327px;
+  height: 122px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+  margin-top: 0;
+  margin-bottom: 65px;
+}
+
+#contact-form {
+  width: 375px;
+  flex-direction: column;
+  align-items: center;
+  padding-left: 48px;
+}
+
+#name {
+  width: 327px;
+  height: 48px;
+  outline: none;
+  font-size: 17px;
+  font-weight: 400;
+  color: #172b4d;
+  background-color: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  box-sizing: border-box;
+  margin-bottom: 20px;
+  padding: 0 16px;
+}
+
+#email {
+  width: 327px;
+  height: 48px;
+  outline: none;
+  font-size: 17px;
+  font-weight: 400;
+  color: #172b4d;
+  background-color: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  box-sizing: border-box;
+  margin-bottom: 20px;
+  padding: 0 16px;
+}
+
+#message {
+  font-family: "Poppins", sans-serif;
+  font-size: 15px;
+  line-height: 24px;
+  height: 170px;
+  padding: 15px 16px;
+  border-radius: 8px;
+  width: 327px;
+  box-sizing: border-box;
+}
+
+.enabled {
+  width: 140px;
+  height: 48px;
+  cursor: pointer;
+  border: none;
+  font-size: 17px;
+  font-weight: 500;
+  letter-spacing: 0.03em;
+  line-height: 24px;
+  color: #6070ff;
+  background: #fff;
+  border-radius: 8px;
+  padding: 12px 16px 12px 16px;
+  transition: all 0.5s ease-in-out;
+}
+
+.enabled:hover {
+  background-color: #0c63e7;
+  color: #fff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+}
+
+.bottom-line {
+  width: 134px;
+  height: 5px;
+  position: relative;
+  top: 20px;
+}
+
+@media (min-width: 768px) {
+  .header {
+    width: 100vw;
+    position: fixed;
+  }
+
+  .mobile-nav {
+    display: none;
+  }
+
+  #nav {
+    display: flex;
+    width: 259px;
+    list-style-type: none;
+    align-items: center;
+    padding: 0;
+    gap: 8px;
+    justify-content: space-around;
+    text-decoration: none;
+  }
+
+  #nav li a {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-size: 15px;
+    line-height: 20px;
+    text-decoration: none;
+    font-weight: 700;
+    color: #344563;
+  }
+
+  #nav li a:hover {
+    text-decoration: underline;
+    text-decoration-color: #6070ff;
+    text-decoration-thickness: 3px;
+  }
+
+  .head-section {
+    width: 100%;
+    height: 846px;
+    background: #fff url(../assets/bg.svg) 100% 73px no-repeat;
+    background-size: contain;
+    border-bottom-left-radius: 150px;
+  }
```

Comment:
- [ ] I noticed you forgot to change the background image in the desktop version, so kindly add it. 👌 remember that According to the requirements You must replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 💯

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/179210219-5019533d-609e-4c6a-a254-5b7c6ca74aea.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/179210299-9aadfed3-8822-4eb2-bd81-d13c3cef7042.PNG" width="500" /> |